projects/gnosis-protocol/index.js

```diff
@@ -0,0 +1,90 @@
+/*==================================================
+  Modules
+==================================================*/
+
+const { api: { abi, util }, util: utilBase } = require('../../sdk');
+
+/*==================================================
+  Settings
+==================================================*/
+
+const START_BLOCK = 9340147;
+const PROTOCOL_ADDRESS = '0x6F400810b62df8E13fded51bE75fF5393eaa841F';
+/* 
+* Tokens on GP to ignore
+[ 
+    0x7cac16770dd5f2a59859a395a492224f05a846b6: "oETH Put $200 29/05/2020",
+]
+*/
+const TOKENS_TO_IGNORE = new Set(['0x7cac16770dd5f2a59859a395a492224f05a846b6']);
```

Comment:
Small concern here.
What will happen when we don't blacklist this kind of token?
Since we cannot prevent people from listing tokens in GP, can some actor (intentionally or not) add a token that might crash GP listing on DeFi Pulse?

Reply:
@bh2smith @fleupold any thoughts here?

Reply:
Unfortunately, this is a tough one. I am not overly familiar with the DeFi pulse integration requirements, but if this file is meant to be a static template then I don't see many options apart from manually updating this ignore list whenever necessary.

Considering that it is DeFi Pulse, one would expect that this is an issue for everyone already listed in their stats aggregator! 😃

Reply:
What happens in case we don't blacklist them? Does it cause issues? If so I think we need to use a whitelist instead.

Reply:
Where does it break? Only tests? [here?](https://github.com/ConcourseOpen/DeFi-Pulse-Adapters/blob/master/testing/test-run.js#L66)
If so, then `symbolLengthLimit = 6` is a rather arbitrary limit and should probably be changed

Reply:
🕵️ to unblock us we could also make sure to blacklist all tokens that have symbol lengths > 6, but I agree that this seems to be an arbitrary limitation.

Reply:
I can spell `SCAM` and `FAKE` in only 4 characters 😄! But more seriously, this bandaid solution would at work for the time being.